great_expectations/dataset/dataset.py

```diff
@@ -486,6 +491,160 @@ def expect_table_columns_to_match_ordered_list(
                 }
             }
 
+    # noinspection PyUnusedLocal
+    @DocInherit
+    @DataAsset.expectation(['min_value', 'max_value'])
+    def expect_table_column_count_to_be_between(
+        self,
+        min_value=None, max_value=None,
+        strict_min=False, strict_max=False,
+        result_format=None, include_config=False, catch_exceptions=None,
+        meta=None,
+    ):
+        """Expect the number of columns to be between two values.
+
+        expect_table_column_count_to_be_between is a :func:`expectation \
+        <great_expectations.data_asset.data_asset.DataAsset.expectation>`, not a
+        ``column_map_expectation`` or ``column_aggregate_expectation``.
+
+        Keyword Args:
+            min_value (int or None): \
+                The minimum number of columns, inclusive unless strict_min=True.
+            max_value (int or None): \
+                The maximum number of columns, inclusive unless strict_max=True.
+            strict_min (boolean):
+                If True, the table column count must be strictly larger than min_value.
+            strict_max (boolean):
+                If True, the table column count be strictly smaller than max_value.
```

Comment:
I don't think this expectation should include strict_min or strict_max since column counts must be integers

Reply:
These arguments are also present for `expect_table_row_count_to_be_between`. Want me to remove the arguments from both?

Reply:
Yes please -- good catch.

Reply:
I found a couple other integer-based expectations with these args, so I removed it in all of those cases.